.env.example

```diff
@@ -1 +1,4 @@
-PORT=<The port you want to run it on>
\ No newline at end of file
+PORT=<The port you want to run it on>
+DEV_DATABASE_URL=postgres://postgres:yourpassword@yourhost:yourport/database-name
+TEST_DATABASE_URL=postgres://postgres:yourpassword@yourhost:yourport/database-name
+DATABASE_URL=postgres://postgres:yourpassword@yourhost:yourport/database-name
```

Comment:
Remember that I told you this should be empty. Heroku will populate the value when in production.